airbyte-config/init/src/main/resources/config/STANDARD_DESTINATION_DEFINITION/079d5540-f236-4294-ba7c-ade8fd918496.json

```
@@ -0,0 +1,7 @@
+{
+  "destinationDefinitionId": "079d5540-f236-4294-ba7c-ade8fd918496",
+  "name": "BigQuery de-normalized",
```

```suggestion
  "name": "BigQuery (Typed Struct)",
```
I think this display name is a little bit clearer? Fine with me if you want to stick with denormalized but if you do it should be one word and I'd suggest putting it in parens. so `BigQuery (Denormalized)`.